client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	Consultant: i18n.translate( 'Consultant' ),
+	Consulting: i18n.translate( 'Consulting' ),
+	'Consulting agency': i18n.translate( 'Consulting agency' ),
+	'Consulting business': i18n.translate( 'Consulting business' ),
+	'Consulting engineering': i18n.translate( 'Consulting engineering' ),
+	Consumables: i18n.translate( 'Consumables' ),
+	'Content creation': i18n.translate( 'Content creation' ),
+	Cooking: i18n.translate( 'Cooking' ),
+	Copywriting: i18n.translate( 'Copywriting' ),
+	'Corporate services': i18n.translate( 'Corporate services' ),
+	'Corrosion engineering': i18n.translate( 'Corrosion engineering' ),
+	Cosmetics: i18n.translate( 'Cosmetics' ),
+	Cosplay: i18n.translate( 'Cosplay' ),
+	Councelling: i18n.translate( 'Councelling' ),
+	Counseling: i18n.translate( 'Counseling' ),
+	'Counseling services': i18n.translate( 'Counseling services' ),
+	'Craft beer': i18n.translate( 'Craft beer' ),
+	Crafting: i18n.translate( 'Crafting' ),
+	'Creative writing': i18n.translate( 'Creative writing' ),
+	'Credit repair': i18n.translate( 'Credit repair' ),
+	'Crime prevention': i18n.translate( 'Crime prevention' ),
+	Crossfit: i18n.translate( 'Crossfit' ),
+	'Crowd funding': i18n.translate( 'Crowd funding' ),
+	'Crypto currency': i18n.translate( 'Crypto currency' ),
+	Culture: i18n.translate( 'Culture' ),
+	Cupcakes: i18n.translate( 'Cupcakes' ),
+	'Current affairs': i18n.translate( 'Current affairs' ),
+	Cycling: i18n.translate( 'Cycling' ),
+	DIY: i18n.translate( 'DIY' ),
+	DJ: i18n.translate( 'DJ' ),
+	DVDs: i18n.translate( 'DVDs' ),
+	'Daily life': i18n.translate( 'Daily life' ),
+	Data: i18n.translate( 'Data' ),
+	'Data visualization': i18n.translate( 'Data visualization' ),
+	Dating: i18n.translate( 'Dating' ),
+	Daycare: i18n.translate( 'Daycare' ),
+	Deals: i18n.translate( 'Deals' ),
+	Depression: i18n.translate( 'Depression' ),
+	Design: i18n.translate( 'Design' ),
+	'Design portfolio': i18n.translate( 'Design portfolio' ),
+	Dessert: i18n.translate( 'Dessert' ),
+	Devops: i18n.translate( 'Devops' ),
+	'Diagnostic machine design': i18n.translate( 'Diagnostic machine design' ),
+	Diary: i18n.translate( 'Diary' ),
+	'Digital design': i18n.translate( 'Digital design' ),
+	'Digital portfolio': i18n.translate( 'Digital portfolio' ),
+	Digitization: i18n.translate( 'Digitization' ),
+	Disease: i18n.translate( 'Disease' ),
+	'Dog boarding kennel': i18n.translate( 'Dog boarding kennel' ),
+	'Dog grooming': i18n.translate( 'Dog grooming' ),
+	Dogs: i18n.translate( 'Dogs' ),
+	'Domestic violence prevention': i18n.translate( 'Domestic violence prevention' ),
+	'Drawing portfolio': i18n.translate( 'Drawing portfolio' ),
+	Drawings: i18n.translate( 'Drawings' ),
+	'Dream coaching': i18n.translate( 'Dream coaching' ),
+	'Dream interpretations': i18n.translate( 'Dream interpretations' ),
+	Dreams: i18n.translate( 'Dreams' ),
+	Dresses: i18n.translate( 'Dresses' ),
+	Drinks: i18n.translate( 'Drinks' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Drink' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).